src/pss/client.js

```diff
@@ -461,7 +482,10 @@ export const addContactRequest = async (
   payload: ContactRequestPayload,
 ) => {
   const contact = {
-    profile: payload.profile,
+    profile: {
+      ...payload.profile,
+      hasStake: true,
```

Comment:
Any reason `hasStake` is set to `true` here without actually checking the stake?

Reply:
As this is from when you receive a contract request I thought we could presume the user has a stake to be able to make the request in the first place?

Reply:
I don't think we should trust peers to provide this information, if they are using the same version of the server that works, but for example a contact request sent from the current version of onyx-server would make this code accept contacts even when they don't have a stake.